packages/sanity/src/cli/actions/exec/configClient.ts

```diff
@@ -1,5 +1,6 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import {getCliClient} from 'sanity/cli'
```

Comment:
Can this be imported from `@sanity/cli` directly?

Reply:
Yep, good catch.